tests/integration/test_vulnerability_detector/test_feeds/alas/test_extra_tags_alas_feed.py

```diff
@@ -87,9 +138,54 @@ def modify_feed(test_values, get_configuration, request):
     vd.clean_vuln_and_sys_programs_tables()
 
 
-def test_no_feed_changes(copy_files, clean_vuln_tables, after_execution_clean_vuln_tables_function,  get_configuration, configure_environment, daemons_handler,
-                         file_monitoring):
-    """Check if the feed is imported successfully by default."""
+def test_no_feed_changes(copy_files, clean_vuln_tables, after_execution_clean_vuln_tables_function,  get_configuration,
+                         configure_environment, daemons_handler, file_monitoring):
+    '''
+    description: Check if the feed is imported successfully by default. To do this, a vulnerability event is expected
+                 and check if the number of vulnerabilities inserted in the VULNERABILITIES table of CVE DB is the
+                 expected.
+
+    wazuh_min_version: 4.2.0
```

Comment:
The Amazon Linux feeds are not available with `4.2.0`, it should be `4.3.0`

```suggestion
    wazuh_min_version: 4.3.0
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/commit/c02bcd31ed6d3d0f57d519bdc2051b8eafeede60#:~:text=c02bcd31ed6d3d0f57d519bdc2051b8eafeede60)